contracts/StablecoinConverter.sol

```diff
@@ -11,7 +11,7 @@ contract StablecoinConverter is EpochTokenLocker {
     using BytesLib for bytes32;
     using BytesLib for bytes;
 
-    uint constant MAX_UINT128 = 2**128 - 1;
+    uint constant private MAX_UINT128 = 2**128 - 1;
```

Comment:
Does making this private improve the readability? Does it save gas?

Reply:
it is better to always declare the visibility!